src/cpp/desktop/DesktopWebProfile.cpp

```diff
@@ -24,6 +24,8 @@ namespace desktop {
 
 namespace {
 
+QWebEngineUrlRequestInfo::ResourceType s_urlRequestInfoResourceType;
```

Comment:
Should this be a static? (can't recall if we have a single WebProfile instance but IIRC there's one per window so each would need its own copy of this)

Reply:
Good point; each WebPage gets their own WebProfile so this should likely be a member variable. I'll fix that up.